src/entities/file.ts

```diff
@@ -24,6 +24,10 @@ export class File implements FileInterface {
     this.character_count = data.character_count;
   }
 
+  hasData() {
+    return this.file_name && this.file_path ? true : false;
```

Comment:
do we need this to be a ternary?

Wouldn't `this.file_name && this.file_path` return something truthy?

Reply:
yeah, it would return undefined since they're non boolean types and that will be enough to validate truthy. i'll update